src/publish.proj

```diff
@@ -69,6 +116,50 @@
                     ManifestCommit="$(ManifestCommit)" />
   </Target>
 
+  <Target Name="PublishTestsToAzureBlobFeed" AfterTargets="PublishToAzureBlobFeed">
```

Comment:
The `AfterTargets` here is concerning. This is creating an implicit coupling that generally confusing to debug. I would update the place where `PublishToAzureBlobFeed` is called and also call `PublishTestsToAzureBlobFeed` instead of the chaining.

Reply:
I agree the coupling is confusing.  The issue here, is that `PublishToAzureBlobFeed` is invoked directly by the VSTS build step, e.g., line 207 in `buildpipeline/DotNet-Trusted-Publish.json`.  This means I would need to change the entry point or the build pipeline.  The latter of which is difficult to test locally.